olm/README.md

```diff
@@ -1,90 +1,112 @@
-- `-b <branch>`: the branch name (mandatory) in the (stackable forked) openshift-certified-operators repository.
-- `-o <operator-name>`: the operator name (mandatory) e.g. airflow-operator.
-- `-d <deploy>`: optional flag for catalog deployment.
+- `-c <manifest folder>`: the output folder for the manifest files
+- `-o <operator-dir>`: directory of the operator repository
+
+Similarly for the listener operator run:
+
+```bash
+./olm/build-manifests.sh -r 24.11.1 \
+  -c $HOME/repo/stackable/openshift-certified-operators \
+  -o $HOME/repo/stackable/listener-operator
+```
+
+## All Other Operators
 
-The script creates a catalog specific to an operator. A catalog can contain bundles for multiple operators, but a 1:1 deployment makes it easier to deploy and test operators independently. Testing with a deployed catalog is essential as the certification pipeline should only be used for stable operators, and a certified operator can only be changed if a new version is specified.
+```bash
+./olm/build-manifests.py \
+  --openshift-versions 'v4.14-v4.16' \
+  --release 24.11.1 \
+  --skips 24.7.0 \
+  --repo-operator ~/repo/stackable/hbase-operator
+```
+
+See `./olm/build-manifests.py --help` for the description of command line arguments.
+
+# Build and Install Bundles
 
-## Use the CI pipeline
+Operator bundles are needed to test the OLM manifests but *not needed* for the operator certification.
 
-### Testing
+## Build bundles
 
-This should either be called from the stackable-utils root folder, or the `volumeClaimTemplateFile` path should be changed accordingly.
+To build operator bundles run:
 
+```bash
+./olm/build-bundles.sh \
+  -c $HOME/repo/stackable/openshift-certified-operators \
+  -r 24.11.1 \
+  -o listener \
+  -d
 ```
-export GIT_REPO_URL=https://github.com/stackabletech/openshift-certified-operators
-export BUNDLE_PATH=operators/stackable-commons-operator/23.4.1
-export LOCAL_BRANCH=commons-23.4.1
-
-tkn pipeline start operator-ci-pipeline \
-  --namespace stackable-operators \
-  --param git_repo_url=$GIT_REPO_URL \
-  --param git_branch=$LOCAL_BRANCH \
-  --param bundle_path=$BUNDLE_PATH \
-  --param env=prod \
-  --param kubeconfig_secret_name=kubeconfig \
-  --workspace name=pipeline,volumeClaimTemplateFile=olm/templates/workspace-template.yml \
-  --showlog
+
+Where:
+- `-r <release>`: the release number (mandatory). This must be a semver-compatible value to patch-level e.g. 23.1.0.
+- `-c <manifest folder>`: the folder with the input OLM manifests for the bundle
+- `-o <operator-name>`: the operator name (mandatory) e.g. "airflow"
+- `-d`: Optional. Deploy the bundle. Default: false.
+N.B. This action will push the bundles to `oci.stackable.tech` and requires that the user be logged in first. This can be done by copying the CLI token from the Harbor UI once you are logged in there (see under "Profile"), and then using is as the password when prompted on entering `docker login oci.stackabe.tech`.
```

Comment:
```suggestion

N.B. This action will push the bundles to `oci.stackable.tech` and requires that the user be logged in first. This can be done by copying the CLI token from the Harbor UI once you are logged in there (see under "Profile"), and then using this as the password when prompted on entering `docker login oci.stackabe.tech`.
```

Reply:
"is" --> "this"